presto-tests/src/test/java/com/facebook/presto/tests/TestDistributedSortQueries.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.Session;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.SystemSessionProperties.DISTRIBUTED_SORT;
+import static com.facebook.presto.SystemSessionProperties.REDISTRIBUTE_SORT;
+import static com.facebook.presto.tests.tpch.TpchQueryRunner.createQueryRunner;
+
+public class TestDistributedSortQueries
+        extends AbstractTestQueries
```

Comment:
to me it is an overkill. Lets select queries that are interesting to test and test them for all the combination of session properties.

Ps. I think we need to revisit all the other tests like that like Test*Spill*